ts/src/coinbase.ts

```diff
@@ -2648,8 +2648,7 @@ export default class coinbase extends Exchange {
     }
 
     async findAccountId (code, params = {}) {
-        await this.loadMarkets ();
```

Comment:
not sure we should do this here
![image](https://github.com/user-attachments/assets/15f181da-1a2b-4dba-947f-1db685fe32ca)

Reply:
question here, even if `fetchAccounts` internally use `loadMarkets`, does it matter? what I see, as long as we have safe-thread access to markets loading promise, it shouldn't be a problem. not?

Reply:
ah wait, i missed the point probably, you're right, fAccounts shouldnt be using results from loaded markets. ok, give me a moment to revise them